src/Nancy/NancyEngine.cs

```diff
@@ -19,7 +19,7 @@
     /// <summary>
     /// Default engine for handling Nancy <see cref="Request"/>s.
     /// </summary>
-    public class NancyEngine : INancyEngine
+    public sealed class NancyEngine : INancyEngine

```

Comment:
Either `sealed` or fully implement the dispose pattern? Unless designed for inheritance, I'm on the side of 'sealed'.